src/client/graph.rs

```diff
@@ -82,6 +82,12 @@ where
         Identify { client: self }
     }
 
+    pub fn custom_endpoint(&'a self, custom_endpoint: &str) -> Identify<'a, Client> {
```

Comment:
Looks good. I think at some point it would be nice to provide a setter so that it doesn't need to be entered each time you use the client, but that would require some rework and something thats not necessarily needed right now. 

Is `custom_endpoint` a common name for this? The word endpoint is usually referring to the whole of the URL and not just the host or domain. I could see `custom_host` or something similar. Ehh I guess also if you pass a URL with a path on it that would be an endpoint and its entirely possible. Not sure on this, I guess im fine with it and it can be revisited if needed.

Reply:
@DevLazio I added you as a collaborator to the project if you want to accept. Just don't abuse it if you do accept it. Meaning just try to ask me first about changing something if its something that your sure about. I don't care if you create labels or anything like that, I just care about the project itself and how its kept/managed so as long as your helping with that, you won't hear anything from me.

Reply:
I don't know if it's a common name and as a not native english speaker I can't really give my opinion on this.
I proposed the name because Microsoft labeled it a "[Service root endpoint](https://docs.microsoft.com/en-us/previous-versions/office/office-365-api/api/o365-china-endpoints#service-root-endpoint)".

I tested it with "https://microsoftgraph.chinacloudapi.cn/v1.0" and I can confirme it works. FYI, you need to negociate an Oauth2 token with these parameters :
Access Token URL:
https://login.chinacloudapi.cn/{{TenantID}}/oauth2/v2.0/token
Scope:
https://microsoftgraph.chinacloudapi.cn/.default

Reply:
Makes sense. `custom_endpoint` works. Going by the names that is used by Microsoft for their APIs is the correct approach so 👍

Reply:
Tests are failing cause forks cant look at repo secrets. Gonna merge.

Reply:
> @DevLazio I added you as a collaborator to the project if you want to accept. Just don't abuse it if you do accept it. Meaning just try to ask me first about changing something if its something that your unsure about. I don't care if you create labels or anything like that, I just care about the project itself and how its kept/managed so as long as your helping with that, you won't hear anything from me.

Thanks ! Great powers comes with great responsability. 
I will stick to Issues / PR for now, to be sure to move in the same direction as you.